privaterelay/templates/includes/premium-onboarding.html

```diff
@@ -70,18 +71,27 @@ <h2>{% ftlmsg 'multi-part-onboarding-premium-welcome-subheadline' %}</h2>
         <li class="c-premium-onboarding-step c-premium-onboarding-step-3 {% if user_profile.onboarding_state == 2 %} is-visible {% endif %}">
             <div class="mzp-l-content mzp-t-content-lg mzp-t-content-nospace">
                 <div class="c-premium-onboarding-headlines">
-                    <h1>{% ftlmsg 'multi-part-onboarding-premium-extension-headline' %}</h1>
-                    <h2>{% ftlmsg 'multi-part-onboarding-premium-welcome-subheadline' %}</h2>
+                    <h1>{% ftlmsg 'multi-part-onboarding-reply-headline' %}</h1>
```

Comment:
I think this headline should probably also differ between small and large screens - it currently says "Reply to your emails" on large screens, rather than something about extensions.

Reply:
Done!